<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editar</class>
 <widget class="QDialog" name="Editar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar</string>
  </property>
  <widget class="QLineEdit" name="txtEdPlaca">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Placa...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEdMarca">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Marca...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEdCor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Cor...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEdModelo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Modelo...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salvar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEdPlaca">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Placa</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEdCor">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cor</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEdMarca">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Marca</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEdModelo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modelo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>Editar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
